weaviate/collections/classes/types.py

```diff
@@ -20,7 +20,41 @@ class GeoCoordinate(_WeaviateInput):
     longitude: float
 
     def _to_dict(self) -> Dict[str, float]:
-        return {"latitude": self.latitude, "longitude": self.longitude}
+        return self.model_dump(exclude_none=True)
+
+
+class _PhoneNumberBase(_WeaviateInput):
+    number: str
+
+
+class PhoneNumber(_PhoneNumberBase):
+    """Input for the phone number datatype.
+
+    `default_country` should correspond to the ISO 3166-1 alpha-2 country code.
+    This is used to figure out the correct countryCode and international format if only a national number (e.g. 0123 4567) is provided.
+    """
+
+    default_country: Optional["ISOCountryCode"] = Field(default=None)
+
+    def _to_dict(self) -> Dict[str, str]:
+        out = {"input": self.number}
+        if self.default_country is not None:
+            out["defaultCountry"] = self.default_country
+        return out
+
+
+class _PhoneNumber(_PhoneNumberBase):
+    """Output for the phone number datatype."""
+
+    country_code: int
+    default_country: str
+    international_formatted: str
+    national: int
+    national_formatted: str
+    valid: bool
+
+
+PhoneNumberType: TypeAlias = _PhoneNumber
```

Comment:
why not just call it directly `PhoneNumberType`?

Reply:
The idea here, which I'm planning to replicate across the API, is to have classes that act as returns as private, e.g. `_Object`, exposed to the users as type aliases, e.g. `Object`. In this way, the only exported classes are for use as inputs with the exported types for use as outputs when users type their own code

Reply:
can we have a more descriptive name for the file? return_types or something?

Reply:
Agreed, I'll make it in another PR so as not to collide with other PRs